executor.py

```diff
@@ -110,7 +117,7 @@ def filter(self, parameters: Dict, **kwargs):
         specifications in the `find` method of `DocumentArray` using Weaviate: https://docarray.jina.ai/fundamentals/documentarray/find/#filter-with-query-operators
         :param parameters: parameters of the request
         """
-        return self._index.find(parameters['query'])
+        return self._index.find(parameters.get('filter', None))
```

Comment:
what will happen if filter is not in parameters ?
options are:
* return nothing
* return all
* raise an error

Reply:
For me it should return all like if the filter was None